extras/ams_wrapper/src/api/types.py

```diff
@@ -74,16 +74,13 @@ def as_dict(self):
         }
 
 class SingleClassification:
-    def __init__(self, tag: Tag, attributes: List[Attribute] = None):
-        self.tag = tag
+    def __init__(self, attributes: List[Attribute] = None):
```

Comment:
If we remove tag and classification consists of attributes then we shouldn't allow `None`. 
We would have "empty" object then.

Reply:
Done.